storops/unity/resource/type_resource.py

```diff
@@ -29,7 +32,14 @@ class UnityType(UnityResource):
 
     def _get_raw_resource(self):
         url = '/api/types/{}'.format(self._id)
-        return self._cli.rest_get(url, fields=self._fields)
+        resp = self._cli.rest_get(url, fields=self._fields)
+        try:
+            # The only known exception is UnityResourceNotFoundError
+            resp.raise_if_err()
+        except UnityResourceNotFoundError:
+            # We translate the type not found error as not supported error
+            raise UnityResourceNotSupportedError("Resource is not supported.")
```

Comment:
Better to add the log here.  You know the resource, it was `self._id`.  And it's definitely related to the type.  So you could safely say it in the log that it's the __type__ which is not available.